.circleci/config.yml

```diff
@@ -8,22 +8,45 @@ workflows:
     jobs:
       - build
       - test
+defaults:
+  ruby_image: &ruby_image
+    - image: circleci/ruby:2.5.5-stretch-node
+
+  executor: &executor
+    executor: ruby/default
+
+  restore_cache: &restore_cache
+    keys:
+      - gem-cache-v1-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+      - gem-cache-v1-{{ arch }}-{{ .Branch }}
+      - gem-cache-v1
+
+  install_dependencies: &install_dependencies
+    name: Install dependencies
+    command: |
+      sudo apt-get update
+      gem update --system
+      gem install bundler -v 2.2.15
```

Comment:
Don't hardcode it, please. You can detect bundle version from your `Gemfile.lock`.

Reply:
Thanks!